telegram/ext/utils/webhookhandler.py

```diff
@@ -123,7 +123,7 @@ def _ensure_event_loop(self, force_event_loop: bool = False) -> None:
                 and (
                     isinstance(
                         asyncio.get_event_loop_policy(),
-                        asyncio.WindowsProactorEventLoopPolicy,  # pylint: disable=E1101
+                        asyncio.WindowsProactorEventLoopPolicy,  # pylint: disable  # type: ignore
```

Comment:
mypy doesn't seem to recognize the type ignore thing since the pylint disable is there too. Removing the pylint comment and keeping the type ignore thing raises no errors for me.

Also now mypy complains the same for line 98:
```
- hook id: mypy
- exit code: 1

telegram/ext/utils/webhookhandler.py:98: error: Module has no attribute "ProactorEventLoop"; maybe "SelectorEventLoop" or "AbstractEventLoop"?  [attr-defined]
telegram/ext/utils/webhookhandler.py:126: error: Module has no attribute "WindowsProactorEventLoopPolicy"; maybe "AbstractEventLoopPolicy"?  [attr-defined]
```
```suggestion
                        asyncio.WindowsProactorEventLoopPolicy,  # type: ignore
```

Reply:
> doesn't pylint complain when you remove that comment? 

nope, not on my system anyway.
> does switching the order of the two comments help? 

yep, tried and it works, pushing..